crypto/ube/snapsafe_detect.c

```diff
@@ -10,16 +10,19 @@
 #include <stdlib.h>
 #include <sys/mman.h>
 #include <unistd.h>
-#include "../delocate.h"
+
+#include "../internal.h"
 
 // Snapsafety state
 #define SNAPSAFETY_STATE_FAILED_INITIALISE 0x00
 #define SNAPSAFETY_STATE_SUCCESS_INITIALISE 0x01
 #define SNAPSAFETY_STATE_NOT_SUPPORTED 0x02
 
-DEFINE_STATIC_ONCE(aws_snapsafe_init)
-DEFINE_BSS_GET(volatile uint32_t *, sgc_addr)
-DEFINE_BSS_GET(int, snapsafety_state)
+static CRYPTO_once_t aws_snapsafe_init = CRYPTO_ONCE_INIT;
+
+static volatile uint32_t *sgc_addr = NULL;
```

Comment:
Curious, what's `sgc` stand for?

Reply:
huh should probably have been `sgn`. I think it stands for "snapsafe generation counter". Synonym terminology I used at some point

Reply:
Changed to `sgn`